packages/transition-backend/src/tasks/disseminationBlocks/importDisseminationBlocks.ts

```diff
@@ -0,0 +1,384 @@
+    }
+
+    private async promptOverwriteDataSource(dataSourceName: string): Promise<boolean> {
+        const answers = await inquirer.prompt([
+            {
+                type: 'list',
+                name: 'overwrite',
+                message: `${dataSourceName} already exists. Do you want to re-create the places in this data source?`,
+                choices: [
+                    { name: 'Yes', value: true },
+                    { name: 'No', value: false }
+                ]
+            }
+        ]);
+        return answers.overwrite;
+    }
+
+    // This function is meant to work with the GML file found here: https://www12.statcan.gc.ca/census-recensement/2021/geo/sip-pis/boundary-limites/index2021-eng.cfm?year=21
+    private async importBoundaries(boundariesFile: string | undefined, dataSourceId: string): Promise<void> {
+        let file = boundariesFile;
+        if (file === undefined) {
+            // Request the file from the user
+            const answers = await inquirer.prompt([
+                {
+                    type: 'file-tree-selection',
+                    name: 'zoneFilePath',
+                    message: 'Please select the dissemination blocks boundaries file to import',
+                    pageSize: 20
+                }
+            ]);
+            file = answers.zoneFilePath;
+        }
+
+        if (typeof file !== 'string') {
+            throw new Error('File is undefined');
+        }
+
+        if (!fileManager.fileExistsAbsolute(file)) {
+            throw new Error(`The requested file ${file} does not exist`);
+        }
+
+        const parser = xmlReader.create({ stream: true, parentNodes: false });
+        const readStream = fs.createReadStream(file);
+        let i = 0;
+
+        const batchSize = 1000;
+        let batch: ZoneAttributes[] = [];
+        let srIdArray: string[] = [];
+        let geographyArray: string[] = [];
+
+        console.log('Parsing through GML FILE:');
+
+        parser.on('tag:gml:featureMember', (feature: XmlNode) => {
+            try {
+                const fme = feature.children[0];
+
+                // The id of the dissemination block.
+                const dbuid = (fme.children.find((element) => element.name === 'fme:DBUID') as XmlNode).children[0]
+                    .value;
+
+                const multiCoorList: string[] = [];
+
+                // The id of the coordinate system used. Necessary to later convert into the coordinates used by Transition.
+                let srId: string = '';
+
+                // The GML file uses two types of geometry for the blocks: simple surfaces and multi surfaces, with slightly different structures that need their own parsing logic
+                // Most of them are simple surfaces, with multi surfaces having multiple non connected polygons, such as a block that includes islands
+                const surfaceProperty = fme.children.find((element) => element.name === 'gml:surfaceProperty');
```

Comment:
All those fields: `gml:surfaceProperty`, `gml:LinearRing`, are they part of the GML standard or Stat Can's?